helpers/templates.json

```diff
@@ -1,17 +1,17 @@
 {
   "confirm_email": {
-    "from": "SteemConnect <noreply@steemconnect.com>",
-    "subject": "Welcome to SteemConnect!",
-    "text": "Thanks for signing up to SteemConnect, we are excited to have you on board!\nTo complete and confirm your registration to SteemConnect, you need to verify your email address.\n\n{url}\n\nIf you encounter any problem using the service, contact us and a member of our team will help you.\n\nCheers,\nSteemConnect"
+    "from": "Steemit <noreply@steemit.com>",
+    "subject": "Please confirm your email",
+    "text": "On Steemit, you get paid to post and share good content. But first, let's finish setting up your account.\n\nPlease confirm your email address by clicking on this link:\n\n{url}\n\nCheers\nYour friends at Steemit"
   },
   "create_account": {
-    "from": "SteemConnect <noreply@steemconnect.com>",
-    "subject" : "Create your account",
-    "text": "Your information have been verified, please click on the link to continue. \n\n{url}\n\n Cheers,\nSteemConnect"
+    "from": "Steemit <noreply@steemit.com>",
+    "subject" : "Welcome to Steemit",
+    "text": "Hi again,\n\nYour Steemit account is approved, so you're all set up and ready to go! Get started by checking out the trending page!\n\n{url}\n\n Cheers,\nYour friends at Steemit"
```

Comment:
This url does not go to the trending page but to the last step of the process where the account is created